### PR TITLE
Assing styles to form

### PR DESCRIPTION
**Pull request by Kosichenko**

### CODE REVIEW

File: slidez-fe-core/src/containers/poll-editor/PollEditor.scss [hunk 1]
```diff
@@ -1,47 +1,86 @@
...
 .app {
-    margin: 2rem;
+    margin: 0.5rem 1rem 1rem 1rem;
 }
 
-.text-input,
-.poll-name {
-    margin-bottom: 20px;
+.text-input {
+    font-family: "Rubik";
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/poll-editor/PollEditor.scss]: We apply font-family globally, so or make so in .app class or remove, as it should be Rubik by default

File: slidez-fe-core/src/containers/poll-editor/PollEditor.scss [hunk 2]
```diff
@@ -1,47 +1,86 @@
...
         color: $main-color;
         margin-right: 5px;
     }
 }
-
-input {
-    padding: 0.5rem;
-    font-size: 16px;
+.poll-name {
+    font-family: "Rubik";
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/poll-editor/PollEditor.scss]: +1, everywhere

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx [hunk 3]
```diff
@@ -43,26 +43,72 @@ const PollEditor: React.FC<PollEditorProps> = ({ presentationId }) => {
...
+                        xmlns='http://www.w3.org/2000/svg'
+                    >
+                        <g opacity='0.5'>
+                            <path
+                                fill='#979797'
+                                stroke='#979797'
+                                d='M15.1983 16.8019C15.3153 16.6848 15.3154 16.4951 15.1985 16.3779L11.0416 12.2119C10.9247 12.0948 10.9247 11.9052 11.0416 11.7881L15.1985 7.62213C15.3154 7.50494 15.3153 7.31517 15.1983 7.1981L14.2123 6.21213C14.0951 6.09497 13.9052 6.09497 13.788 6.21213L8.21229 11.7879C8.09513 11.905 8.09513 12.095 8.21229 12.2121L13.788 17.7879C13.9052 17.905 14.0951 17.905 14.2123 17.7879L15.1983 16.8019Z'
+                            />
+                        </g>
+                    </svg>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: Have a look how we insert svg as images and make the same (you can check dashboard sidebar)

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx [hunk 4]
```diff
@@ -43,26 +43,72 @@ const PollEditor: React.FC<PollEditorProps> = ({ presentationId }) => {
...
+                <span className='poll-name'>Live poll</span>
+                <svg
+                    width='12'
+                    height='8'
+                    viewBox='0 0 12 8'
+                    fill='#59D0A5'
+                    xmlns='http://www.w3.org/2000/svg'
+                >
+                    <path d='M10.59 0.589844L6 5.16984L1.41 0.589844L0 1.99984L6 7.99984L12 1.99984L10.59 0.589844Z' />
+                </svg>
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: +1

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx [hunk 5]
```diff
@@ -73,19 +119,24 @@ const PollEditor: React.FC<PollEditorProps> = ({ presentationId }) => {
...
-                                                            arrayHelpers.remove(
-                                                                index
-                                                            )
-                                                        }
+                                                        onClick={() => {
+                                                            if (index > 0) {
+                                                                arrayHelpers.remove(
+                                                                    index
+                                                                )
+                                                            }
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: Move it to function
const toggleRemoveClick = () => { if (index > 0) ....}

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: slidez-fe-core/src/containers/poll-editor/PollEditor.scss (hunk 1 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/poll-editor/PollEditor.scss]: Removed

File: slidez-fe-core/src/containers/poll-editor/PollEditor.scss (hunk 2 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/poll-editor/PollEditor.scss]: Removed

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx (hunk 5 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: Done

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx (hunk 3 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: Done

File: slidez-fe-core/src/containers/poll-editor/PollEditor.tsx (hunk 4 above)
Comment (Kosichenko) [reply slidez-fe-core/src/containers/poll-editor/PollEditor.tsx]: Done

**Review by alex-tiutiunnyk — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**